### PR TITLE
Add Radius Bank

### PR DESCRIPTION
**Pull request by mmomjian**
No documentation available, image: 
image

### CODE REVIEW

**Comment by phallobst** [comment]
The logo is very hard to see against the light green background when the middle part is transparent. Maybe this one with an opaque white center is better:
radiusbank

File: _data/banking.yml [hunk 1]
```diff
@@ -1150,6 +1150,15 @@ websites:
...
 
+  - name: Radius Bank
+    url: https://radiusbank.com
+    img: radiusbank.png
+    tfa:
+      - sms
+      - phone
+      - email
+      - totp
+
```
Comment (phallobst) [inline _data/banking.yml:1163]:
```suggestion
    regions:
      - us

```

**Comment by mmomjian** [comment]
Thank you, updated

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please use this SVG icon instead:
Radius Bank SVG

**Comment by mmomjian** [comment]
done @ApCoder123

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**